### ISSUE TITLE
Can't get MnasNet mean & stdev for input normalization: get correct classes but incorrect scores.

### ISSUE THREAD

**Issue by Jonarod** [tensorflow/tensorflow#31464]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow):** Stock example `label_image.py`
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04):** Linux Ubuntu 18.04
- **TensorFlow installed from (source or binary):** Tensorflow Lite installed from source.
- **TensorFlow version (use command below):** v2.0.0-beta0-16-g1d91213fe7 2.0.0-beta1
- **Python version:** 3.6.7
- **Bazel version (if compiling from source):** using Make, not bazel
- **GCC/Compiler version (if compiling from source):** gcc
- **CUDA/cuDNN version:** no CUDA
- **GPU model and memory:** no GPU

### Describe the current behavior
The docs are quite unclear about which normalization technique and constants are used for inputs. But generally speaking, popular models like `mobilenet` use an input range of `[0,1]`.
This is the case for most models hosted [here](https://www.tensorflow.org/lite/guide/hosted_models), except `MnasNet`. For some reasons, I cannot properly use inference with the `MnasNet_1.0_224` `.tflite` model using input ranges of `[0,1]`. For instance, after normalizing my input from 0 to 1, I do get the correct output class but get an incorrect output score. In fact, **normalizing RGB channels from `[0,255]` to `[0,1]` produces scores above 100% (sometimes 900%...) using TFLite's `MnasNet_1.0_224`**.

Before writing this issue, I have randomly tested other input ranges like `[-1,1]` and `[-127,128]` but with no luck either. In PyTorch docs there are some references of using a normalization per channels like `[0.485*255, 0.456*255, 0.456*255]` for the mean and `[0.229*255, 0.224*255, 0.225*255]` for the standard deviation (for each R,G and B channel respectively), but once again to no avail: **correct class with unscaled scores**.

### Describe the expected behavior

More than an expected behavior I would expect some documentation around models. Recently I could see a new convention to ship models with a text file informing input and output channels names which I think is great. However, IMHO it should also ship with `mean` and `stdev` used for input normalization. Moreover, and here I am deviating from the original issue briefly, I would love to see these information **coded** in some constant variables like `input.get_normalization_range()` or something like that (just my 2cts).

Now regarding that specific issue, is anyone able to tell if that is an MnasNet bug not respecting classic ranges, or a deliberate training-related optimization ?
And, of course, I'd love to see a code or pseudo-code to get my input right using MnasNet :smile: 

### Code to reproduce the issue

Just download the `MnasNet_1.0_224.tflite` model here: [mnasnet_1.0_224_09_07_2018.tgz](https://storage.cloud.google.com/download.tensorflow.org/models/tflite/mnasnet_1.0_224_09_07_2018.tgz)
Extract it.
Then, copy the stock python script `label_image.py` here: [label_image.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/examples/python/label_image.py)
Then get a random image to run inference.

Finally just run inference but providing mnasnet model instead of default one:
```
python3 label_image.py --model_file mnasnet_1.0_224/mnasnet_1.0_224.tflite
```

**Comment by amahendrakar** [comment]
@Jonarod,
Is this still an issue? Could you please update TensorFlow to v2.3 and check if you are still facing the same issue. Thanks!
